src/MigrationTools.Host/StartupService.cs

```diff
@@ -35,11 +35,11 @@ public void RunStartupLogic(string[] args)
             {
                 Version latestVersion = _detectVersionService.GetLatestVersion();
 
-                _logger.LogInformation("Latest version detected as {@Version}", latestVersion);
+                _logger.LogInformation($"Latest version detected as {{{nameof(latestVersion)}}}", latestVersion);
```

Comment:
If you use $ then the version will not be passed to AI. Or are you doing something clever here with multiple {}

Reply:
The multiple {} means we are still doing {latestVersion} but if we rename latestVersion it will drop the new name into the string. @ovebastiansen can correct me if I'm wrong. Were we logging like this before because we wanted each part of the version info in the logs as a data point? Then maybe we should just write to console differently

Reply:
We just want a single data point for version... I think we need:

`_logger.LogInformation("Latest version detected as {Version}", latestVersion.toString());`

Reply:
Log.Warning("Disk quota {Quota} MB exceeded by {User}", quota, user);
The usage of the {Quota} name has nothing to do with the variable quota, so the usage of nameof is not needed in a refactoring perspective.